spring-cloud-gcp-starters/spring-cloud-gcp-starter-config/src/main/resources/META-INF/spring.factories

```diff
@@ -1,4 +1,2 @@
 org.springframework.cloud.bootstrap.BootstrapConfiguration=\
-org.springframework.cloud.gcp.config.autoconfig.GcpConfigAutoConfiguration,\
-org.springframework.cloud.gcp.core.autoconfig.GcpContextAutoConfiguration,\
-org.springframework.cloud.gcp.config.GoogleConfigPropertySourceLocator
+org.springframework.cloud.gcp.config.GoogleConfigPropertySourceLocator
```

Comment:
New line in the end of file.
The IDEA can be configured to do that automatically.